### PR TITLE
Merch dashboard orders

### LINKED ISSUE #31: User Story 40, Merchant Dashboard displays Orders
```
As a merchant
When I visit my dashboard ("/dashboard")
If any users have pending orders containing items I sell
Then I see a list of these orders.
Each order listed includes the following information:
- the ID of the order, which is a link to the order show page ("/dashboard/orders/15")
- the date the order was made
- the total quantity of my items in the order
- the total value of my items for that order
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #33

child of #79

depends on #32

### PR DESCRIPTION
**Pull request by aprildagonese**
Closes #31 
- changes _orderslist partial to work from @orders collection instead of @user.orders
- fixes fallout from above change
- adds test and functionality for User Story 40:
As a merchant
When I visit my dashboard ("/dashboard")
If any users have pending orders containing items I sell
Then I see a list of these orders.
Each order listed includes the following information:
- the ID of the order, which is a link to the order show page ("/dashboard/orders/15")
- the date the order was made
- the total quantity of my items in the order
- the total value of my items for that order

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
 put '/profile', to: 'users#update'
 get '/profile/orders', to: 'user/orders#index'
 patch '/profile/orders', to: 'user/orders#update'
-get '/profile/orders/:id', to: 'user/orders#show'
+get '/profile/orders/:id', to: 'user/orders#show', as: 'profile/order'
```
Comment (PeregrineReed) [inline config/routes.rb]: this should be profile_order

**Comment by PeregrineReed** [comment]
@nagerz brought up the question today of if each order item and each order need a status to be fulfilled, I just reread through all the merchant user stories and we definitely need it built like that.

On card #7, User Story 61 a merchant can fulfill an individual item for an order. When all order items are fulfilled then the order status should be 'shipped'. In order to keep building out the merchant dashboard we'll have to implement these things.

@smthom05 I'm gonna tag you too, just to keep you in the loop.
